packages/workspace/src/migrations/update-10-3-0/add-cli-dependency.ts

```diff
@@ -0,0 +1,9 @@
+import { chain, Rule } from '@angular-devkit/schematics';
+import { formatFiles, addDepsToPackageJson } from '@nrwl/workspace';
+
+export default function update(): Rule {
+  return chain([
+    addDepsToPackageJson({}, { '@nrwl/cli': '10.3.0' }),
```

Comment:
Normally, I would say to use hard coded versions in migrations but because this is a nrwl package, we should use `nxVersion` so that they get the same version of `@nrwl/cli` installed as `@nrwl/workspace`.

```suggestion
    addDepsToPackageJson({}, { '@nrwl/cli': nxVersion }),
```

```ts
import { nxVersion } from '../../../src/utils/versions';
```